Tools/clinic/clinic.py

```diff
@@ -3035,6 +3058,9 @@ class CConverter(metaclass=CConverterAutoRegister):
     # Only set by self_converter.
     signature_name: str | None = None
 
+    # Optional #include "name"  // reason
```

Comment:
Just to distinguish which part is which in the tuple:
```suggestion
    # Optional (#include "name",  // reason)
```